Userland/Games/GameOfLife/BoardWidget.cpp

```diff
@@ -138,12 +166,32 @@ void BoardWidget::paint_event(GUI::PaintEvent& event)
 
 void BoardWidget::mousedown_event(GUI::MouseEvent& event)
 {
-    set_toggling_cells(true);
-    auto row_and_column = get_row_and_column_for_point(event.x(), event.y());
-    if (!row_and_column.has_value())
-        return;
-    auto [row, column] = row_and_column.value();
-    toggle_cell(row, column);
+    if (event.button() != GUI::MouseButton::Right) {
```

Comment:
I don't think we need to care about middle clicks either, so this should be fine:

```suggestion
    if (event.button() == GUI::MouseButton::Left) {
```